examples/applications/plot_stock_market.py

```diff
@@ -131,9 +161,7 @@
     'CSCO': 'Cisco',
     'TXN': 'Texas instruments',
```

Comment:
while we're at it, the `i` should probably be uppercase, same for the 'e' in American Express, and I think "Mc Donalds" should be "McDonald's".